### PR TITLE
Added a simple Issue Template.

### PR DESCRIPTION
**Pull request by Teodor92**
For more info on Issue and PR templates - [https://help.github.com/articles/creating-an-issue-template-for-your-repository/](https://help.github.com/articles/creating-an-issue-template-for-your-repository/)

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+# Steps to Reproduce
+
+1.
+1.
+1.
+
+# Specifications
+
+- Platform: [What platform are you using? softuni.bg / judge.softuni.bg / other]
```
Comment (rozay) [inline ISSUE_TEMPLATE.md]: SoftUni (https://softuni.org), Software University (https://softuni.bg), SoftUni Digital (https://digital.softuni.bg), SoftUni Creative (https://creative.softuni.bg), SoftUni Kids (https://kids.softuni.bg), SoftUni Judge (https://judge.softuni.bg), SoftUni Test System (https://softuni.bg:8080)

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+- Browser used: [What browser are you using?]
```
Comment (rozay) [inline ISSUE_TEMPLATE.md]: Maybe add username too

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+
+# Expected Behavior
+
+# Actual Behavior
+
+# Steps to Reproduce
+
+1.
+1.
```
Comment (rozay) [inline ISSUE_TEMPLATE.md]: 2. 😄

Comment (Teodor92) [reply ISSUE_TEMPLATE.md]: Markdown, does not care but fixed it nonetheless 😄

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (Teodor92) [reply ISSUE_TEMPLATE.md]: 1ww2er

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (Teodor92) [reply ISSUE_TEMPLATE.md]: Done.

**Review by rozay — APPROVED** [review]

**Comment by Teodor92** [comment]
please-merge-me

**Comment by rozay** [comment]
@Teodor92 Merged and thank you :)

**Pull request merged**